articles/active-learning/index.md

```diff
@@ -0,0 +1,150 @@
+![hero](/engineering-education/active-learning/hero.jpg)
+
+[Image Source](https://images.unsplash.com/photo-1546410531-bb4caa6b424d?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=1051&q=80)
+
+By nature, machine learning models are data-hungry. They require a lot of data to make decisions. However, always having high quality labeled data to train models is often a costly task. An answer to this would be to use an approach that can reduce the training data needed to train a model. Active learning is such an approach.
+
+### Contents
+
+1. Introduction to active learning
+
+2. Steps to use active learning on unlabelled data
+
+3. Three active learning frameworks
+
+4. Query strategies
+
+### Prerequisites
+
+This article requires an understanding of supervised learning, in addition to machine learning in general. This [article](https://www.section.io/engineering-education/supervised-learning-algorithms/) may be of help with the mentioned prerequisites.
```

Comment:
Rewrite section links as /engineering-education/.... no need of https://www.section.io/